### DISCUSSION TITLE
Wiki information high up in the README

### DISCUSSION THREAD

**Discussion by RaZ0rr-Two** [hrsh7th/nvim-cmp#548] [General]
Maybe put mappings and other wiki information high up in the readme. These are very resourceful (atleast for me).

I think it might be better to put [this one](https://github.com/hrsh7th/nvim-cmp/wiki/Example-mappings) above [this info](https://github.com/hrsh7th/nvim-cmp#how-to-set-up-mappings)

Right now it is only shown way [near the bottom](https://github.com/hrsh7th/nvim-cmp#how-to-set-up-mappings).

Also, [this one](https://github.com/hrsh7th/nvim-cmp/wiki/Menu-Appearance) is super cool. Thank you

**Comment by hrsh7th** [comment]
Thank you. PR are welcome :)

**Reply by RaZ0rr-Two** [reply]
alright :+1:

**Comment by RaZ0rr-Two** [comment]
On a off-topic note, what tool do you use to get that neovim gif in the README?

Is it [peek](https://github.com/phw/peek) ?

**Comment by hrsh7th** [comment]
I was using kap on Mac OS X.
